packages/fx-core/tests/core/other.test.ts

```diff
@@ -3,62 +3,82 @@
+import { assert } from "chai";
 import { randomAppName } from "./utils";
 import sinon from "sinon";
 import fs from "fs-extra";
 import os from "os";
 import * as path from "path";
 import { defaultSolutionLoader } from "../../src/core/loader";
-import { FetchSampleError, NoneFxError, ProjectFolderExistError, ReadFileError, TaskNotSupportError, WriteFileError } from "../../src/core/error";
+import {
+  FetchSampleError,
+  NoneFxError,
+  ProjectFolderExistError,
+  ReadFileError,
+  TaskNotSupportError,
+  WriteFileError,
+} from "../../src/core/error";
+import mockedEnv from "mocked-env";
+import { isFeatureFlagEnabled } from "../../src/common/tools";
 
 describe("Other test case", () => {
-
   const sandbox = sinon.createSandbox();
 
   afterEach(() => {
     sandbox.restore();
   });
   it("question: QuestionAppName validation", async () => {
-    const inputs:Inputs = {platform: Platform.VSCode};
+    const inputs: Inputs = { platform: Platform.VSCode };
     const folder = os.tmpdir();
     let appName = "1234";
     inputs.folder = folder;
- 
-    let validRes = await (QuestionAppName.validation as FuncValidation<string>).validFunc(appName, inputs);
 
-    assert.isTrue(validRes === "Application name must start with a letter and can only contain letters and digits.");
+    let validRes = await (QuestionAppName.validation as FuncValidation<string>).validFunc(
+      appName,
+      inputs
+    );
+
+    assert.isTrue(
+      validRes ===
+        "Application name must start with a letter and can only contain letters and digits."
+    );
 
     appName = randomAppName();
-    let projectPath = path.resolve(folder, appName);
-    
+    const projectPath = path.resolve(folder, appName);
+
     sandbox.stub<any, any>(fs, "pathExists").withArgs(projectPath).resolves(true);
-    
-    validRes = await (QuestionAppName.validation as FuncValidation<string>).validFunc(appName, inputs);
+
+    validRes = await (QuestionAppName.validation as FuncValidation<string>).validFunc(
+      appName,
+      inputs
+    );
     assert.isTrue(validRes === `Path exists: ${projectPath}. Select a different application name.`);
 
     sandbox.restore();
     sandbox.stub<any, any>(fs, "pathExists").withArgs(projectPath).resolves(false);
-    
-    validRes = await (QuestionAppName.validation as FuncValidation<string>).validFunc(appName, inputs);
-    assert.isTrue(validRes === undefined);
 
+    validRes = await (QuestionAppName.validation as FuncValidation<string>).validFunc(
+      appName,
+      inputs
+    );
+    assert.isTrue(validRes === undefined);
   });
 
   it("loader: DefaultSolutionLoader", async () => {
-    const inputs:Inputs = {platform: Platform.VSCode};
+    const inputs: Inputs = { platform: Platform.VSCode };
     const solution = await defaultSolutionLoader.loadSolution(inputs);
-    assert.isTrue(solution.name  === "fx-solution-azure");
+    assert.isTrue(solution.name === "fx-solution-azure");
```

Comment:
maybe assert.strictEqual is better :-)

Reply:
leave to test case owner to make the decision... I don't know why this file is not formatted before.